pandas/core/reshape/encoding.py

```diff
@@ -316,3 +318,198 @@ def get_empty_frame(data) -> DataFrame:
             dummy_mat = dummy_mat[:, 1:]
             dummy_cols = dummy_cols[1:]
         return DataFrame(dummy_mat, index=index, columns=dummy_cols)
+
+
+def from_dummies(
+    data: DataFrame,
+    sep: None | str = None,
+    default_category: None | Hashable | dict[str, Hashable] = None,
+) -> DataFrame:
+    """
+    Create a categorical ``DataFrame`` from a ``DataFrame`` of dummy variables.
+
+    Inverts the operation performed by :func:``~pandas.get_dummies``.
```

Comment:
you just need single backticks if you already have `:func:`
```suggestion
    Inverts the operation performed by :func:`~pandas.get_dummies`.
```